config/errros.go

```diff
@@ -0,0 +1,8 @@
+package config
```

Comment:
filename has a typo in it.

Reply:
ohh that was always bad ha, let me fix it

Reply:
done